cmd/vic-init/ops.go

```diff
@@ -83,6 +86,20 @@ func (t *operations) SetHostname(name string, aliases ...string) error {
 	return t.BaseOperations.SetHostname(aliases[0])
 }
 
+func (t *operations) SetupFirewall() error {
+	// get the public interface name
+	l, err := netlink.LinkByName(publicIfaceName)
```

Comment:
Anywhere close enough to be discovered by other people. In this case it might be even public I think.

I am not asking to do this for this review, but if you add TODO comment it would be great.